.github/workflows/build.yml

```diff
@@ -107,6 +107,8 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - run: npm run build-storybook
+        env:
+          NODE_OPTIONS: --openssl-legacy-provider
```

Comment:
Aside: I have a memory of having to do this as a workaround on a repo (at my previous company) until we managed to upgrade node, but I can't remember the versions involved.